code/api/src/modules/user/model.js

```diff
@@ -22,6 +22,10 @@ module.exports = function(sequelize, DataTypes) {
 
   User.associate = function(models) {
     User.hasMany(models.Subscription)
+    User.belongsTo(models.Style, {
```

Comment:
We will also want to remove this association here as well if we don't end up going with the relationship route